### PR TITLE
feat: adding CNAME support

### PR DESCRIPTION
**Pull request by Ani1357**
## Checklist

- [ ] Architecture Design Records have been added as `adr/*.md` and appended to list in `adr/_index.md`, if applicable.
- [ ] The `values-schema.yaml` file and `test/**` fixtures have been updated to reflect code changes, if applicable.
- [ ] The OpenApi Schema from redkubes/otomi-api project is compatible with definitions from `values-schema.yaml` file, if applicable.
- [ ] Helm releases are meeting otomi's baseline security policies, if applicable.
- [ ] Helm chart and helmfile changes are tested against upgrade scenario, if applicable.

### CODE REVIEW

File: values-schema.yaml [hunk 1]
```diff
@@ -1121,6 +1121,18 @@ definitions:
...
+        description: Will configure additional host(CNAME) for the service.
+        type: boolean
+        default: false
+      cname:
+        properties:
+          domain:
+            description: CNAME of the service.
+            type: string
+          tlsSecretName:
+            description: Kubernetes secret name of type TLS.
```
Comment (j-zimnowoda) [inline values-schema.yaml:1134]: Kubernetes secret name of type TLS (not required if the tlsPass flag is set to true).

**Review by j-zimnowoda — COMMENTED** [review]
It looks good. I am just missing some test values in the `test/fixtures` dir.
Is it possible to automate deployment of a service with CNAME for the testing purpose?

File: values-schema.yaml (hunk 1 above)
Comment (Ani1357) [reply values-schema.yaml:1134]: added cname config for team-demo using "tslPass: true" so that we dont have to provide a tls secret for it to work

**Comment by Ani1357** [comment]
> 

added cname config for team-demo using "tslPass: true" so that we dont have to provide a tls secret for it to work

**Review by j-zimnowoda — APPROVED** [review]

**Pull request merged**